tutorials/tour/2017-02-25-packages-and-imports

```diff
@@ -0,0 +1,85 @@
+---
+layout: tutorial
+title: Packages and Imports
+
+disqus: true
+
+tutorial: scala-tour
+categories: tour
+num: 4
+next-page:
+previous-page: named-parameters
+prerequisite-knowledge: basics, unified-types, classes
+---
+# Packages and Imports
+Scala uses packages to create namespaces which allow you to modularize programs.
+
+## Creating a package
+Packages are created by declaring one or more package names at the top of a Scala file.
+
+```
+package users
+
+class User
+```
+One convention is to name the package the same as the directory containing the Scala file. The directory structure of an sbt project for `package users` might look like this:
```

Comment:
You should probably mention that scala doesn't care about file layout at all. This is just a convention. However some tools might care (I think IntelliJ does).